extensions/VisualEditor/modules/ve-mw/dm/nodes/ve.dm.MWTransclusionNode.js

```diff
@@ -78,7 +78,7 @@ ve.dm.MWTransclusionNode.static.toDataElement = function ( domElements, converte
 		mwDataJSON = domElements[0].getAttribute( 'data-mw' ),
 		mwData = mwDataJSON ? JSON.parse( mwDataJSON ) : {},
 		isInline = this.isHybridInline( domElements, converter ),
-		type = isInline ? 'mwTransclusionInline' : 'mwTransclusionBlock';
+		type = isInline ? 'mwTransclusionInline' : this.name;
```

Comment:
we cannot do like ths, because `this.name` will not be set to `mwTransclusionInline` yet... in this place, all elements despite the `wikiaInfoboxTransclusionBlock` ( because it was already assigned here: https://github.com/Wikia/app/compare/DAT-ve-infobox-edit#diff-47b93bf8eb1c0ca927e2b65ab5b9f234R31 ) will have the type 'mw:Transclusion'. I'll create a new function to handle this and add the @description. Thank you!